modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -1532,18 +1532,24 @@
      },
      "ACTIONS": {
        "SPLIT": "Split",
-       "CUT":   "Cut",
-       "REPLAY_SEGMENT": "Replay segment",
-       "REPLAY_END_OF_SEGMENT": "End of segment",
+       "SPLIT_TOOLTIP": "Create a new trim mark at the current cursor position.",
+       "CUT":   "Hide/Show",
+       "CUT_TOOLTIP":   "Hide or show the selected part of the video.",
+       "REPLAY_SEGMENT": "Play Start of Segment",
+       "REPLAY_SEGMENT_TOOLTIP": "Play the first seconds of the current segment.",
+       "REPLAY_END_OF_SEGMENT": "Play End of segment",
+       "REPLAY_END_OF_SEGMENT_TOOLTIP": "Play a few seconds at the end of the curent segment.",
        "REPLAY_SEGMENT_WITH_PREROLL": "Check transition",
```

Comment:
Transition.
Also maybe "Play Transition" for consistency?

Reply:
fixed